### PR TITLE
add ellipse fit

### PR DESCRIPTION
**Pull request by CoriPegliasco**
- minor english modifs

### CODE REVIEW

**Comment by AntSimi** [comment]
Lint found some incorrect things

File: examples/16_network/pet_ioannou_2017_case.py [hunk 1]
```diff
@@ -188,3 +188,47 @@ def update_axes(ax, mappable=None):
 m = close_to_i3.scatter_timeline(ax, "shape_error_e", vmin=14, vmax=70, **kw)
 cb = update_axes(ax, m["scatter"])
 cb.set_label("Effective shape error")
+
```
Comment (AntSimi) [inline examples/16_network/pet_ioannou_2017_case.py:191]: Maybe add some extra comment :)

**Review by AntSimi — CHANGES_REQUESTED** [review]
Correct error raise by lint and add some extra comment about fitted ellipse

**Comment by CoriPegliasco** [comment]
Hope it works know!

**Review by AntSimi — APPROVED** [review]

**Pull request merged**